spyder/plugins/projects/plugin.py

```diff
@@ -385,6 +391,16 @@ def clear_recent_projects(self):
         self.recent_projects = []
         self.setup_menu_actions()
 
+    def change_max_recent_projects(self):
+        "Change max recent projects entries"""

```

Comment:
```suggestion
        "Change max recent projects entries."""
```

We use a period because docstrings are sentences ;-)